src/js/helpers/awardAmountChartHelper.js

```diff
@@ -233,7 +233,7 @@ export const buildGrantsDirectOtherProps = (awardType, awardAmounts, hasOutlays,
 };
 
 
-export const buildLoanProps = (awardAmounts, hasOutlays, showFilecCovid, awardType, fileCType) => {
+export const buildLoanProps = (awardAmounts, hasOutlays, showFilecCovid, awardType) => {
```

Comment:
are all of these arguments needed here?

Reply:
i removed the unused props from here and from where this function is called